platforms/minigame/common/engine/cache-manager.js

```diff
@@ -198,12 +199,7 @@ const cacheManager = {
         this._write();
         function deferredDelete () {
             const item = caches.pop();
-            if (self._isZipFile(item.originUrl)) {
-                rmdirSync(item.url, true);
-                self._deleteFileCB();
-            } else {
-                deleteFile(item.url, self._deleteFileCB.bind(self));
-            }
+            self._removePathOrFile(item.originUrl, item.url);
```

Comment:
Why call `item.originalUrl` `url` and call `item.url`  `path`?
And why is the method called `_removePathOrFile(url, path)`? The method name `_removePathOrFile` doesn't say any about `url`, and `PathOrFile`, `(url, path)` are mismatch. So `File` is for `url` argument or `path` argument? I guess you mean `path` argument, in that case, Is `File` for `url` argument? and the order ?